### PR TITLE
fixed menu button hover styles

### PR DESCRIPTION
**Pull request by Shiho317**
Hi.
I fixed style of menu button when hovered.
And also I fixed code for working function correctly. So I think its working well for menu button bar is closed when you put menu button.
Please check this code and run your local too for making sure.
Thank you.

### CODE REVIEW

File: components/menu.module.css [hunk 1]
```diff
@@ -101,6 +101,10 @@
   border-bottom: 1px solid white;
 }
 
+.menu li a:hover {
+  color: white;
+}
+
```
Comment (shiv-chan) [inline components/menu.module.css]:
```suggestion
}

.menu li a {
  border-bottom: 0.5rem;
}

.menu li a:hover {
  color: white;
  border-bottom: 1px solid white;
}

```
The underline of each menu item is still longer than it needs to be, so I suggest fixing it this way.
Please check it on your own once again 🙏

**Review by shiv-chan — CHANGES_REQUESTED** [review]
Thank you for working on this, @Shiho317!
There's still one thing to fix, so please check the comment 🙏 
Thanks!

File: components/menu.module.css (hunk 1 above)
Comment (Shiho317) [reply components/menu.module.css]: Hi Kaho! Thank you for reviewing. I fixed underline length. please check review again.

**Review by shiv-chan — APPROVED** [review]

**Comment by shiv-chan** [comment]
It looks good to me now. Thanks, @Shiho317.

**Review by bluwakpa — APPROVED** [review]
Thank you for fixing the hover menu styles! I verified it works locally.

**Pull request merged**